### PR TITLE
#1572 - Fixes navbar-left paddings

### LINKED ISSUE #1572: Adjust screen for Navbar
The Issue

Navbar looks fine on Desktop, but on smaller screens, padding-left is 15px on the left-navbar when it should be 23px as per Zeplin, and there's no space above the entire navbar. It just doesn't look good having the logo touching the top of the screen, no space in-between.

Suggestion to Fix

Adjust padding-left on left-navbar
If we can't keep the same height value for some reason then maybe:

Add padding-top on navbar
Add the same amount in margin-top of the "Help Board" bar-tab which is below
(idk if this is the tidy way to do this, but yeah)

Edit: Same space for the top should be in the bottom!

Front end, front end clean up, mobile 

P4 level

### PR DESCRIPTION
**Pull request by guvarallo**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s):_#1572

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [ ] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

@mvijays this fixes #1572

### CODE REVIEW

File: client/src/components/Header.js [hunk 1]
```diff
@@ -37,6 +37,12 @@ const StyledNavBar = styled(NavBar)`
   .am-navbar-title {
     display: none;
   }
+  .am-navbar-left {
+    @media screen and (max-width: ${mq.phone.wide.maxWidth}) {
+      padding-left: 23px;
```
Comment (idjevm) [inline client/src/components/Header.js]: Please use `rem` and no `px`. Update this to `2.3` rem.

**Review by idjevm — CHANGES_REQUESTED** [review]
I haven't tested on the platform but codewise, I left some comment

**Review by jessica-lau — APPROVED** [review]
Tested on iPhone XS Max and Samsung Galaxy S9 - LGTM. Thanks!

image

image

**Review by nadinejuraschek — APPROVED** [review]
lgtm, thanks!

**Review by idjevm — APPROVED** [review]
LGTM! Thanks @guvarallo !!

**Review by rabbi08 — COMMENTED** [review]
Tested on following devices:
- Google Pixel 3A : 
image

- Galaxy Note 9
image

Thanks

**Pull request merged**